OnboardIos/OnboardIos/Modules/ViewController.m

```diff
@@ -47,20 +52,55 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 {
     static NSString *CellIdentifer = @"CellIdentifier";
     UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:CellIdentifer];
-    
     // Using a cell identifier will allow your app to reuse cells as they come and go from the screen.
     if (cell == nil)
     {
-        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:CellIdentifer];
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:CellIdentifer];
     }
     
-    // Deciding which data to put into this particular cell.
-    // If it the first row, the data input will be "Data1" from the array.
     NSUInteger row = [indexPath row];
     NSDictionary *dict = [[Users list:@1] objectAtIndex:row];
+
+    
+    //informacao de cada linha
     cell.textLabel.text = [NSString stringWithFormat:@"%@ %@", dict[@"first_name"], dict[@"last_name"]];
```

Comment:
comments that identify logic sections normally indicates that the portion of the code under it meaning can be separated in its own method